soh/soh/SohGui/SohMenuRandomizer.cpp

```diff
@@ -71,11 +74,13 @@ void SohMenu::AddMenuRandomizer() {
             "so you never know what you're getting."));
     AddWidget(path, "Simpler Boss Soul Models", WIDGET_CVAR_CHECKBOX)
         .CVar(CVAR_RANDOMIZER_ENHANCEMENT("SimplerBossSoulModels"))
+        .RaceDisable(false)
         .Options(CheckboxOptions().Tooltip(
             "When shuffling boss souls, they'll appear as a simpler model instead of showing the boss' models."
             "This might make boss souls more distinguishable from a distance, and can help with performance."));
     AddWidget(path, "Skip Get Item Animations", WIDGET_CVAR_COMBOBOX)
         .CVar(CVAR_RANDOMIZER_ENHANCEMENT("TimeSavers.SkipGetItemAnimation"))
+        .RaceDisable(false)
```

Comment:
Should probably be disabled for Races, absolutely can make a huge difference in speed.